scripts/generate_errors.pl

```diff
@@ -48,7 +48,7 @@
                             CAMELLIA CCM CHACHA20 CHACHAPOLY CMAC CTR_DRBG DES
                             ENTROPY ERROR GCM HKDF HMAC_DRBG MD2 MD4 MD5
                             NET OID PADLOCK PBKDF2 PLATFORM POLY1305 RIPEMD160
-                            SHA1 SHA256 SHA512 THREADING XTEA );
+                            SHA1 SHA256 SHA512 SM3 THREADING XTEA );
 my @high_level_modules = qw( CIPHER DHM ECP MD
```

Comment:
Please rebase on top of `development` again to have a recent version of `tests/scripts/depends-hashes.pl`, and add an entry for SM3.

Reply:
Hi @gilles-peskine-arm,
I have yet to rebase. I am tempted to rebase only once the CI test is passing, unless you think rebasing will help CI test?

Reply:
CI is passing now :+1:

Reply:
Hurray! Maybe we can even unblock merging by the end of the year ;-)
What about this last error though?:
continuous-integration/jenkins/pr-merge — This commit cannot be built    Details
@oss-mbed-tls-bot PR-4492-merge TLS Testing — Failures: ABI-API-checking    Details